vo_models/vodataservice/models.py

```diff
@@ -0,0 +1,308 @@
+"""Pydantic-xml models for VODataService types
+
+TODO: This is an incomplete spec, covering only elements needed for VOSITables
```

Comment:
Should there be an issue / ticket about this?

Reply:
Added issue https://github.com/spacetelescope/vo-models/issues/17